File: src/main/java/com/hackerrank/introduction/javaloops1/Solution.java
package com.hackerrank.introduction.javaloops1;

import java.util.Scanner;

public class Solution {

  public static void main(String[] args) {
    final Scanner sc = new Scanner(System.in);
    final int n = sc.nextInt();
    for (int i = 1; i <= 10; i++) {
      System.out.printf("%d x %d = %d\n", n, i, n * i);
    }
  }
}

File: src/main/kotlin/com/hackerrank/introduction/javaloops1/Solution.kt
package com.hackerrank.introduction.javaloops1

import java.util.*

fun main(args: Array<String>) {
  val sc = Scanner(System.`in`)
  val n = sc.nextInt()
  for (i in 1..10) {
    System.out.printf("%d x %d = %d\n", n, i, n * i)
  }
}

